.golangci.yaml

```diff
@@ -5,8 +5,8 @@ linters:
   disable-all: true
   enable:
     - asciicheck
-    - deadcode
-    - depguard
+    # - deadcode
+    # - depguard
```

Comment:
Why are we disabling `depguard`?

Reply:
It is a mistake: with new version of golangci-lint it emitted all kind of errors. It turned out that I just needed to add the relevant configuration in `.golangci.yaml`. Fixed.